x/lazyledgerapp/types/payformessage.go

```diff
@@ -0,0 +1,294 @@
+package types
+
+import (
+	"crypto/sha256"
+	"errors"
+	fmt "fmt"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lazyledger/lazyledger-core/crypto/merkle"
+	core "github.com/lazyledger/lazyledger-core/types"
+	"github.com/lazyledger/nmt"
+)
+
+const (
+	TypeMsgPayforMessage                   = "payformessage"
+	TypeSignedTransactionDataPayForMessage = "signedtransactiondatapayformessage"
+	ShareSize                              = core.ShareSize
+	SquareSize                             = 64
```

Comment:
Is there a reason you 1) don't use 128 for square size or 2) use `code.MaxSquareSize`? https://github.com/lazyledger/lazyledger-core/blob/908fe8ac2a43ce0bd372be4938ad25ebd79384fe/types/consts.go#L23

Reply:
Nope, no reason. You're right, it does make sense to use the max. Will update.

Reply:
resolves in https://github.com/lazyledger/lazyledger-app/pull/21/commits/cd89d19e5cac52a2d45db7454bba2c0cd8424423